opm/input/eclipse/Schedule/Well/Well.hpp

```diff
@@ -273,8 +280,15 @@ class Well {
 
         // this is used to check whether the specified control mode is an effective history matching production mode
         static bool effectiveHistoryProductionControl(ProducerCMode cmode);
-        void handleWCONPROD( const std::optional<VFPProdTable::ALQ_TYPE>& alq_type, const UnitSystem& unit_system, const std::string& well, const DeckRecord& record);
-        void handleWCONHIST( const std::optional<VFPProdTable::ALQ_TYPE>& alq_type, const UnitSystem& unit_system, const DeckRecord& record);
+        void handleWCONPROD(const std::optional<VFPProdTable::ALQ_TYPE>& alq_type,
+                            const double bhp_def,
+                            const UnitSystem& unit_system,
+                            const std::string& well,
+                            const DeckRecord& record);
+        void handleWCONHIST(const std::optional<VFPProdTable::ALQ_TYPE>& alq_type,
+                            const double bhp_def,
+                            const UnitSystem& unit_system,
+                            const DeckRecord& record);
```

Comment:
Same here, please.  A comment regarding the expected unit conventions for `bhp_def`.

Reply:
i've added doxy to all methods. turns out there are 3 that eats SI and one that eats input units. To make it all SI I'd have to pass a unit system that was not previously passed so I opted to keep it as-was.